extensions/wikia/AnalyticsEngine/AnalyticsProviderComscore.php

```diff
@@ -12,14 +12,14 @@ function getSetupHtml($params=array()){
 	function trackEvent($event, $eventDetails=array()){
 		switch ($event){
 		  case AnalyticsEngine::EVENT_PAGEVIEW :
-			  if ($this->getC7Value()) {
+			  if (static::getC7Value()) {
```

Comment:
`static` is more flexible and preferred. You can use it in child classes when inheriting methods or attributes.